packages/elements/src/tree-select/index.ts

```diff
@@ -423,7 +423,7 @@ export class TreeSelect extends ComboBox<TreeSelectDataItem> {
     };
     this.queryItems((item, composer): boolean => {
       const hasChildren = composer.getItemChildren(item);
-      if (hasChildren.length) {
+      if (hasChildren.length && this.mode === TreeManagerMode.RELATIONAL) {
```

Comment:
This change makes the `expanded` and `expandable` in the `memo` incorrect, I think you can fix this bug by separating `else if` to the new `if` statement.